t/print_errors.t

```diff
@@ -0,0 +1,76 @@
+use strict;
+use warnings;
+use lib 't/lib';
+use PkgConfig::Capture;
+use Test::More;
+use PkgConfig;
+
+note "DEFAULT_SEARCH_PATH = $_" for @PkgConfig::DEFAULT_SEARCH_PATH;
+
+# adjust to test with real pkg-config
+my @pkg_config = ( $^X, $INC{'PkgConfig.pm'} );
+#my @pkg_config = ( 'pkg-config' );
+
+
+my $nonexistent_lib = 'rubbish-no-exists';
+
+subtest 'ppkg-config --print-errors with non-existent lib' => sub {
```

Comment:
Can you also make a subtest for the default behavior?

Reply:
Done